app/assets/javascripts/location.coffee

```diff
@@ -0,0 +1,84 @@
+$('.address_auto_complete_field, .edit_address_auto_complete_field').livequery ->
+  window.placeAutocomplete = new google.maps.places.Autocomplete($(this)[0], types: [ 'geocode' ])
+  placeAutocomplete.addListener('place_changed', fillInAddress);
+
+componentForm =
+  location_administrative_area_level_2: 'short_name'
+  location_locality: 'long_name'
+  location_administrative_area_level_1: 'short_name'
+  location_country: 'long_name'
+  location_sublocality_level_1: 'long_name'
+  location_postal_code: 'short_name'
+
+fillInAddress = ->
+  place = placeAutocomplete.getPlace()
+  $.each componentForm, (index, value) ->
+    document.getElementById(index).value = ''
```

Comment:
@deepali1-aconnexion why clear JS is used here? Why not `$('#' + index)...` as below in this method?